Automattic/jetpack
# Search E2E: fix search E2E for block themes

#### Changes proposed in this Pull Request:
- Changed page ready selector to `.wp-block-search__button`, which stops relying on theme content
- Run `auto_config` command to add Search widget/input before E2E tests
- Clear search plan info before test, just to make tests more robust

#### Jetpack product discussion
p1643195266035100-jetpack-search-team-slack

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Follow [instructions](https://github.com/Automattic/jetpack/blob/master/projects/plugins/jetpack/tests/e2e/README.md) to set up E2E env
* Upgrade WP to `5.9`
* Change the theme to 2022 or other block themes
* Ensure all five E2E tests pass: `pnpm test-e2e -- search`
![image](https://user-images.githubusercontent.com/1425433/152089570-64dbaa98-02a1-4beb-b98d-4cb7a0cfdcc8.png)
